src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -502,7 +502,7 @@ public function dropColumn($tableName, $columnName)
      * @param string $tableName Table Name
      * @return array
      */
-    protected function getIndexes($tableName)
+    public function getIndexes($tableName)
```

Comment:
needed for the unitests -> to get the current indexes and not the ones in the runtime within the table object